rules/prefer-math-min-max.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const MESSAGE_ID = 'prefer-math-min-max';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` to simplify ternary expressions.',
+};
+
+/**
+@param {import('eslint').Rule.RuleContext} context
+@param {import('estree').ConditionalExpression} node
+@param {string} method
+*/
+function reportPreferMathMinOrMax(context, node, left, right, method) {
+	const {sourceCode} = context;
+
+	context.report({
```

Comment:
Please use `return` or `yield` to avoid `content.report` in all your PRs.

Reply:
I'm not sure why return a report is better than `context.report({})`, since I see it used elsewhere.

And function calls have intelligence with LSP, which is less error-prone than returning a value.

Anyway, update in 7cd3d39d86adde4186f5a2ae2a14fc222fbec36e